### PR TITLE
[ADD] keyword for Pa02

### PR DESCRIPTION
**Pull request by af-pwo**
[ADD] test case for Pa02
[ADD] Jsons for Pa02 and PaC02
[FIX] Specified info in /tests/readme

### CODE REVIEW

File: src/test/resources/Observation/create-pac02.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      {
+        "system": "http://loinc.org",
+        "code": "2019-8",
+        "display": "Carbon dioxide [Partial pressure] in Arterial blood"
+      }
+    ],
+    "text": "Carbon dioxide [Partial pressure] in Arterial blood"
+  },
+  "subject": {
+    "reference": "Patient/c786ac3c-0579-4aa2-adda-db784b214ad6"
```
Comment (af-dpe) [inline src/test/resources/Observation/create-pac02.json]: Subject is not correct needs to be fixed

File: src/test/resources/Observation/create-pa02.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "coding": [
+      {
+        "system": "http://loinc.org",
+        "code": "2703-7",
+        "display": "Oxygen [Partial pressure] in Arterial blood"
+      }
+    ],
+    "text": "Oxygen [Partial pressure] in Arterial blood"
+  },
+  "subject": {
```
Comment (af-dpe) [inline src/test/resources/Observation/create-pa02.json]: Subject is not correct needs to be fixed

**Review by af-dpe — CHANGES_REQUESTED** [review]
Fix subject in the jsons

keyword and test case for PaO2 is fine

File: src/test/resources/Observation/create-pac02.json (hunk 1 above)
Comment (af-pwo) [reply src/test/resources/Observation/create-pac02.json]: Adjusted to:
```
"subject": {
    "identifier": {
      "system": "urn:ietf:rfc:4122",
	  "value": "{{patientId}}"
	}
  }
```

File: src/test/resources/Observation/create-pa02.json (hunk 2 above)
Comment (af-pwo) [reply src/test/resources/Observation/create-pa02.json]: Adjusted to 
```
"subject": {
    "identifier": {
      "system": "urn:ietf:rfc:4122",
	  "value": "{{patientId}}"
	}
  }
```

**Pull request merged**